components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2096,6 +2123,48 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
         }
     }
 
+    /**
+     * Parse supported encryption algorithms set and add them to supportedIdTokenEncryptionAlgorithms.
+     * @param algorithms   OMElement of supported algorithms.
+     */
+    private void parseSupportedIdTokenEncryptionAlgorithms(OMElement algorithms) {
+        if (algorithms == null) {
+            return;
+        }
+
+        Iterator iterator = algorithms.getChildrenWithLocalName(ConfigElements.SUPPORTED_IDTOKEN_ENCRYPTION_ALGORITHM);
+        if (iterator != null) {
+            for (; iterator.hasNext(); ) {
+                OMElement algorithm = (OMElement) iterator.next();
+                if (algorithm != null) {
+                    log.info(algorithm.getText());
```

Comment:
is it make sense to just log algorithm.getText()

Reply:
That was mistakenly left there. I'll remove it ASAP.